cmd/main.go

```diff
@@ -92,11 +112,37 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&controller.NATSReconciler{
-		Client:          mgr.GetClient(),
-		Scheme:          mgr.GetScheme(),
-		NATSProvisioner: provisioner.NATSProvisioner{},
-	}).SetupWithManager(mgr); err != nil {
+	// create helmRenderer
+	const repoDir = "/resources/nats"
```

Comment:
`repoDir` -> `resourcesDir`

Reply:
This will be refactored in next [PR](https://github.com/kyma-project/nats-manager/pull/19/files#) as env [config](https://github.com/kyma-project/nats-manager/blob/d4f80667511638151a1d41e14bf676beaa4283b4/pkg/env/config.go).